packages/node_modules/@node-red/editor-client/src/js/ui/view-tools.js

```diff
@@ -176,8 +176,8 @@ RED.view.tools = (function() {
         }
         nodes.forEach(function(n) {
             var modified = false;
-            var oldValue = n.l === undefined?true:n.l;
-            var showLabel = n._def.hasOwnProperty("showLabel")?n._def.showLabel:true;
+            var showLabel = n._def.hasOwnProperty("showLabel") ? n._def.showLabel : true;
```

Comment:
Wouldn't it be better to use the "editor.view.view-node-show-label" parameter as the default value (instead of true) ?